src/Mutator/MutatorResolver.php

```diff
@@ -166,7 +200,7 @@ private static function registerFromClass(
         if ($settings === false) {
             unset($mutators[$mutatorClassName]);
         } else {
-            $mutators[$mutatorClassName] = $settings === true ? [] : (array) $settings;
+            $mutators[$mutatorClassName] = (array) $settings;
```

Comment:
Cast needed?

Reply:
We do a cast in any case, I'm just removing the unnecessary ternary statement

Reply:
Right. It is `false` to `[]` type of cast.